### DISCUSSION TITLE
EXPO AND RXRANGE SETTING NOT WORKING

### DISCUSSION THREAD

**Discussion by FF1970** [iNavFlight/inav#10399] [Q&A]
Hello
i can't set rc yaw expo 0.70 doesn't change anything?
the yaw is too fast
and in the cli rxrange doesn't work?
can you help me under inav 7.1.1 FC F405
thanks

**Comment by MrD-RC** [comment]
Unless this is a 3D plane, you shouldn't need 70% expo. Your throws are most likely way too large.

**Comment by FF1970** [comment]
hello
no it's for an initiation drone you have to be very gentle on the controls
i need a lot of gentleness on the sticks

**Reply by MrD-RC** [reply]
You need to be more specific. Is this fixed wing or multirotor?

**Comment by FF1970** [comment]
It's a 3 inch multirotor

**Reply by MrD-RC** [reply]
If the yaw is too fast, reduce the rates.

**Comment by FF1970** [comment]
I usually use betaflight and I set the expo and rx range easily

**Comment by FF1970** [comment]
![Capture d'écran 2024-10-01 203355](https://github.com/user-attachments/assets/f752dd04-8c60-43e1-9bc1-84d30bf9b318)

**Comment by FF1970** [comment]
hello
i want to change the channel range in the cli with rxrange it doesn't work the values ​​are not supported
example rxrange 2 1295 1700 SAVE nothing
i am INAV 7.1.2
i would like to avoid doing it with the radio

**Reply by sensei-hacker** [reply]
> hello i want to change the channel range in the cli with rxrange it doesn't work the values ​​are not supported example rxrange 2 1295 1700 SAVE nothing i am INAV 7.1.2 i would like to avoid doing it with the radio

What exactly are you *expecting* to happen when you set the range, that isn't happening?

**Comment by FF1970** [comment]
hello
i am waiting for a reduction in the range of use of the sticks
for a specific use

**Reply by sensei-hacker** [reply]
> hello i am waiting for a reduction in the range of use of the sticks for a specific use


rxrange *informs* INAV of the range sent by your radio.
It doesn't *change* what your radio is sending.

What are you actually trying to accomplish - what is your end goal?

**Comment by FF1970** [comment]
https://github.com/iNavFlight/inav/blob/master/docs/Rx.md
**I found a doc**

Receiver Channel Range Configuration.
If you have a transmitter/receiver, that output a non-standard pulse range (i.e. 1070-1930 as some Spektrum receivers) you could use rx channel range configuration to map actual range of your transmitter to 1000-2000 as expected by INAV.

The low and high value of a channel range are often referred to as 'End-points'. e.g. 'End-point adjustments / EPA'.

All attempts should be made to configure your transmitter/receiver to use the range 1000-2000 before using this feature as you will have less preceise control if it is used.

To do this you should figure out what range your transmitter outputs and use these values for rx range configuration. You can do this in a few simple steps:

If you have used rc range configuration previously you should reset it to prevent it from altering rc input. Do so by entering the following command in CLI:

rxrange reset
save
Now reboot your FC, connect the configurator, go to the Receiver tab move sticks on your transmitter and note min and max values of first 4 channels. Take caution as you can accidentally arm your craft. Best way is to move one channel at a time.

Go to CLI and set the min and max values with the following command:

rxrange <channel_number> <min> <max>
For example, if you have the range 1070-1930 for the first channel you should use rxrange 0 1070 1930 in the CLI. Be sure to enter the save command to save the settings.

After configuring channel ranges use the sub-trim on your transmitter to set the middle point of pitch, roll, yaw and throttle.

You can also use rxrange to reverse the direction of an input channel, e.g. rxrange 0 2000 1000.
